book.rb

```diff
@@ -0,0 +1,14 @@
+require './rental'
+class Book
+  attr_accessor :author, :title, :rental
+
+  def initialize(author, title)
+    @author = author
+    @title = title
+    @rental = []
+  end
+
+  def add_rental(_person, date)
+    Rental.new(perason, date, self)
```

Comment:
- [ ] it looks like you made a small typo in this line, it should be `person` instead of `perason`.

Reply:
done